maas/plugins/galera_check.py

```diff
@@ -45,10 +47,12 @@ def generate_query(host, port):
         port = ' -P %s' % port
     else:
         port = ''
-
-    return ('/usr/bin/mysql --defaults-file=/root/.my.cnf'
-            '%s%s -e "SHOW STATUS WHERE Variable_name REGEXP '
-            "'^(wsrep.*|queries)'\"") % (host, port)
+    if output_type == 'status':
+        return ('/usr/bin/mysql --defaults-file=/root/.my.cnf '
+                '%s%s -e "SHOW GLOBAL STATUS"') % (host, port)
+    elif output_type == 'variables':
+        return ('/usr/bin/mysql --defaults-file=/root/.my.cnf '
+                '%s%s -e "SHOW VARIABLES"') % (host, port)
```

Comment:
Why are you not using 'SHOW GLOBAL VARIABLES' here but introducing GLOBAL for 'SHOW STATUS'?